src/test/java/easytests/core/mappers/TesteesMapperTest.java

```diff
@@ -1,116 +1,96 @@
+        final List<TesteeEntity> testeeFixtureEntities = new ArrayList<>();
+        for(Integer index = 0; index < 2; index++){
+            testeeFixtureEntities.add(this.testeesSupport.getEntityFixtureMock(index));
+        }
+        for (TesteeEntity testeeFixtureEntity: testeeFixtureEntities){
+            if (testeeFixtureEntity.getQuizId() == 3){
+                this.testeesSupport.assertEquals(testeeFixtureEntity, testee);
+            }
+        }
     }
 
     @Test
+    //1hr
     public void testInsert() throws Exception{
         final ArgumentCaptor<Integer> id = ArgumentCaptor.forClass(Integer.class);
-        final String firstName = "FirstName";
-        final String lastName = "LastName";
-        final String surname = "Surname";
-        final Integer groupNumber = 307;
-        final Integer quizId = 19;
+        final TesteeEntity testeeAdditionalEntity = this.testeesSupport.getEntityAdditionalMock(0);
+        this.testeesMapper.insert(testeeAdditionalEntity);
 
-        TesteeEntity testeeEntity = Mockito.mock(TesteeEntity.class);
-        Mockito.when(testeeEntity.getFirstName()).thenReturn(firstName);
-        Mockito.when(testeeEntity.getLastName()).thenReturn(lastName);
-        Mockito.when(testeeEntity.getSurname()).thenReturn(surname);
-        Mockito.when(testeeEntity.getGroupNumber()).thenReturn(groupNumber);
-        Mockito.when(testeeEntity.getQuizId()).thenReturn(quizId);
-
-        this.testeesMapper.insert(testeeEntity);
+        verify(testeeAdditionalEntity, times(1)).setId(id.capture());
+        Assert.assertNotNull(id.getValue());
 
-        verify(testeeEntity, times(1)).setId(id.capture());
+        final TesteeEntity testeeInsertedEntity = this.testeesMapper.find(id.getValue());
 
-        Assert.assertNotNull(id.getValue());
+        Assert.assertNotNull(testeeInsertedEntity);
+        this.testeesSupport.assertEqualsWithoutId(testeeAdditionalEntity, testeeInsertedEntity);
 
-        testeeEntity = this.testeesMapper.find(id.getValue());
-        Assert.assertEquals(id.getValue(), testeeEntity.getId());
-        Assert.assertEquals(firstName, testeeEntity.getFirstName());
-        Assert.assertEquals(lastName, testeeEntity.getLastName());
-        Assert.assertEquals(surname, testeeEntity.getSurname());
-        Assert.assertEquals(groupNumber, testeeEntity.getGroupNumber());
-        Assert.assertEquals(quizId, testeeEntity.getQuizId());
     }
 
     @Test
+    // 1hr
     public void testUpdate() throws Exception{
-        final Integer id = 1;
-        final String firstName = "NewFirstName";
-        final String lastName = "NewLastName";
-        final String surname = "NewSurname";
-        final Integer groupNumber = 308;
-        final Integer quizId = 19;
-
-        TesteeEntity testeeEntity = this.testeesMapper.find(id);
+        final TesteeEntity testeeAdditionalEntity = this.testeesSupport.getEntityAdditionalMock(1);
+        final Integer id = testeeAdditionalEntity.getId();
```

Comment:
We could avoid `id` use `testeeAdditionalEntity.getId()` instead

Reply:
Don't quite understand.

Reply:
@janchk  ride of id variable here, just use testeeAdditionalEntity.getId() instead